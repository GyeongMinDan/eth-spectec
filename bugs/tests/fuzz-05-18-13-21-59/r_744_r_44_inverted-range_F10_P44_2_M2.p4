enum bit<16> V {
 b = !..., o = 1 };
