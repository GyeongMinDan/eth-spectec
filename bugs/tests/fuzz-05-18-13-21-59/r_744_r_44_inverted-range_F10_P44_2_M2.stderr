likely_hits/r_744_r_44_inverted-range_F10_P44_2_M2.p4(2): [--Werror=type-error] error: Cannot apply ! to value ... of type ANYTYPE
 b = !..., o = 1 };
      ^^^
likely_hits/r_744_r_44_inverted-range_F10_P44_2_M2.p4(2): [--Werror=type-error] error: Cannot apply ! to value ... of type ANYTYPE
 b = !..., o = 1 };
      ^^^
likely_hits/r_744_r_44_inverted-range_F10_P44_2_M2.p4(2): [--Werror=type-error] error: Error while analyzing !...
 b = !..., o = 1 };
     ^^^^
In file: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172
Compiler Bug: /p4c/frontends/p4/typeChecking/typeChecker.cpp:172: Null srcType

